00_core.ipynb

```diff
@@ -148,7 +148,7 @@
    "outputs": [],
```

Comment:
<p>Careful - don't use <code>[]</code> as a default! https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastkaggle/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Good to know - Thanks!</p><p> </p><p>Fixed.</p>